src/core/infer_request.h

```diff
@@ -201,6 +201,7 @@ class InferenceRequest {
     std::vector<int64_t> shape_with_batch_dim_;
     bool is_shape_tensor_;
     std::shared_ptr<Memory> data_;
+    bool optional_;
```

Comment:
For consistency can you move after is_shape_tensor_ and call it is_optional_tensor_.

Reply:
Actually this variable can be removed, this is a spot I missed after I removed 
 the "optional" getter/setter in `InferenceRequest`. As Triton core doesn't really need this information and the backend can check if a input is optional via the `BackendModel` function